src/utils/settingUtils.ts

```diff
@@ -6,7 +6,13 @@
 import * as vscode from "vscode";
 
 export function isTerminalSetToCloudShell(): boolean {
-    return vscode.workspace.getConfiguration().get("azureTerraform.terminal") === "cloudshell";
+    if (vscode.workspace.getConfiguration().get("azureTerraform.terminal") === "cloudshell") {
```

Comment:
When the extension activates, it checks "isTerminalSetToCloudShell" and shows the warning message. And it will also be showed when user uses vscode commands like `init`/`plan`.

How about updating user's config to `integrated` if it's `cloudshell`. If so, below warning message will be showed only once.